integration-tests/micrometer-native/src/test/java/io/quarkus/it/micrometer/native_mode/NativeMeterRegistriesIT.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.it.micrometer.native_mode;
+
+import static io.restassured.RestAssured.given;
+
+import java.util.List;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.NativeImageTest;
+import io.restassured.response.Response;
+
+@NativeImageTest
+class NativeMeterRegistriesIT {
```

Comment:
This class can extend `NativeMeterRegistriesTest`?

Reply:
It can, sure, but as the results are different in native mode vs. not, it isn't running the same test.

Since there is a general expectation that one will extend the other, I've put that back, but I've also added an @Override annotation to hopefully call more attention to the fact that the native mode test is not the same.